tools/onnx2tensorrt.py

```diff
@@ -0,0 +1,241 @@
+import argparse
+import os
+import os.path as osp
+from typing import Iterable, Optional, Union
+
+import mmcv
+import cv2
+import numpy as np
+import onnxruntime as ort
+import torch
+from mmcv.ops import get_onnxruntime_op_path
+from mmcv.tensorrt import (TRTWraper, is_tensorrt_plugin_loaded, onnx2trt,
+                           save_trt_engine)
+
+from mmedit.datasets.pipelines import Compose
+
+
+def get_GiB(x: int):
+    """return x GiB."""
+    return x * (1 << 30)
+
+
+def _prepare_input_img(config: dict,
+                       shape: Optional[Iterable] = None,
+                       rescale_shape: Optional[Iterable] = None) -> dict:
+    """Prepare the input image
+
+    Args:
+        config (dict): MMCV config, determined by the inpupt config file.
+        shape (Optional[Iterable]): min shape of the input tensor.
+        rescale_shape (Optional[Iterable]): to rescale the shape of the input tensor.
+    
+    Returns:
+        dict: {'imgs': imgs, 'img_metas': img_metas}
+    """
+    # remove alpha from test_pipeline
+    model_type = args.model_type
```

Comment:
Errr... I means, put variables `args` or `args.model_type`  into arguments. We may reuse this function in other scripts.